src/PolyType.TestCases/TestCase.cs

```diff
@@ -12,7 +12,6 @@ namespace PolyType.Tests;
 /// </summary>
 public static class TestCase
 {
-#if NET
```

Comment:
@AArnott notice how nicely the new `ResolveDynamic` method reconciles the netstandard and modern .NET APIs here.